src/lib/blueprint.ts

```diff
@@ -743,6 +743,12 @@ const createResponse = (
   ) {
     const properties = schema.properties
 
+    if (!(responseKey in properties)) {
```

Comment:
Some endpoints are not supposed to have a responseKey (return void). We just need to check that the responseKey is defined then the key is actually in the response.

Reply:
Isn't this handled earlier in code https://github.com/seamapi/blueprint/blob/51872e4796210c17067955594ef8cb4cfc81b6ee/src/lib/blueprint.ts#L705-L710?

Reply:
Ah got it, I misinterpreted this change. Looks correct.